src/app/pages/test-report/test-report.analytics.effects.ts

```diff
@@ -266,8 +266,9 @@ export class TestReportAnalyticsEffects {
         ),
       ),
     )),
-    // eslint-disable-next-line max-len
-    concatMap(([action, tests]: [ReturnType <typeof manoeuvresADIPart2Actions.AddManoeuvreDrivingFault>, TestsModel]) => {
+    concatMap((
+      [action, tests]: [ReturnType <typeof manoeuvresADIPart2Actions.AddManoeuvreDrivingFault>, TestsModel],
```

Comment:
do we need action here?

Reply:
Yes, it is used twice on line 275 ``` ${manoeuvreTypeLabels[action.payload.manoeuvre]} - ${manoeuvreCompetencyLabels[action.payload.competency]}``` . This would also fail via ESLint if it was unused.